### ISSUE TITLE
Things overlay - skip unnecessary  click when adding new thing

### ISSUE THREAD

**Issue by mnalis** [streetcomplete/StreetComplete#5562]
**Use case**

When adding a **new** Thing/Place (via the `+`) button, I'd like to add it more quickly. Current UI requires me to needlessly click on `Select` dropdown list every time (it is needless as I cannot click anything else when adding a **new** thing, but I always must click on it).

**Proposed Solution**
When adding a new thing via `+` button, open the dialog with list of things automatically, instead of requiring user to click on a `Select` dropdown list first. I.e. skip the first picture below and go directly to the second one.

![small_Screenshot_20240331_004653_StreetComplete](https://github.com/streetcomplete/StreetComplete/assets/156656/71f5bf09-668b-44ba-af5f-184c208d0d07) ![small_Screenshot_20240331_004622_StreetComplete](https://github.com/streetcomplete/StreetComplete/assets/156656/2f8ef728-af62-40dc-9cc9-7e19e965a1b8)

That would save a needless click on every add (which would be quite helpful, especially when user is walking along and adding multiple missing things, like benches / streetlamps / trees / ...)

Idea is probably applicable to Places overlay too.

**Comment by matkoniecz** [comment]
See #5020 - I remember also similar for lit quest.

**Comment by mnalis** [comment]
Well, it is somewhat different, as such situation  does not exist in way-based overlays like surface/lit (as SC does not support adding new ways).

For surface/lit overlays that you mention, depending on what element was clicked, user is presented with either with "not mapped" or with actual value, so  **changing the UI depending on specific clicked element** might indeed be seen as confusing.

That is however not the situation when adding _**new**_ POI in Things/Places overlay -- here we are guaranteed that UI will be the **same for _EVERY_ newly added POI**. So we might strive for more ergonomic UI for POI-based overlays, and be consistent there.

**Comment by FloEdelmann** [comment]
I usually have the map centered at my location, rotated in the compass direction and with a slightly tilted view. When I tap the + button, 3D mode is disabled and thus the marker location changes a bit. So my first action after tapping the + button is moving the map so that the marker is at the correct location. Only then do I actually choose the POI type. So immediately opening the list would require me to remember to relocate the marker after the selection and after the enter button has already activated.

**Comment by westnordost** [comment]
See @FloEdelmann , i.e. your suggestion would not be an improvement for all use cases / workflows, also it would be inconsistent to other UI in the app as nowhere else a dialog pops up into your face when opening a quest or overlay form either, sorry. Since you are using SCEE anyway, this could be a setting there 🤷

**Comment by westnordost** [comment]
On the other hand, after the upcoming migration to maplibre this is something that could be revisited, as (for the moment) I disabled 3d-buildings altogether in the Maplibre style because they just don't look that good. So maybe that would change the workflow of @FloEdelmann and others.

**Comment by mnalis** [comment]
Sure, I was unaware of @FloEdelmann issue when I wrote the suggestion. Although I would classify his issue as tedious manual workaround for Tangram-ES bug (where it loses correct location on switching view), and not as a reasonable and expected workflow. :smile_cat:  

I'm glad to hear that Maplibre version will be without 3D; while it might be nice eye candy in other applications (like e.g. routing), I find it brings way too much problems when used in mapping app (e.g. https://github.com/streetcomplete/StreetComplete/discussions/5235, https://github.com/streetcomplete/StreetComplete/issues/1589, https://github.com/streetcomplete/StreetComplete/issues/4563, ...). 
I hope that alone will make @FloEdelmann workflow easier indeed!

I'll revisit the issue when Maplibre becomes available then; if still not wanted, I'd then suggest to SCEE (as you note - it would solve my personal peeve; I just thought it would be useful generally in SC, so I suggested it here first).
